File: find_aperitivo.py
import os

file_path = 'original_clean.js'
search_str = 'Aperitivo'

with open(file_path, 'r', encoding='utf-8') as f:
    content = f.read()
    index = content.find(search_str)
    if index >= 0:
        print(f"Found at {index}")
        print(content[index-200:index+3000])
    else:
        print("Not found")

File: find_aperitivo.js
const fs = require('fs');
const content = fs.readFileSync('original_clean.js', 'utf8');
const searchStr = 'Aperitivo';
const index = content.indexOf(searchStr);
if (index >= 0) {
    console.log(`Found at ${index}`);
    console.log(content.substring(index - 200, index + 3000));
} else {
    console.log('Not found');
}
